debug_toolbar/settings.py

```diff
@@ -159,13 +158,8 @@ def get_panels():
     return PANELS
 
 
-@lru_cache()
-def get_patch_settings():
-    return getattr(settings, 'DEBUG_TOOLBAR_PATCH_SETTINGS', settings.DEBUG)
-
-
-# The following functions can monkey-patch settings automatically. Several
-# imports are placed inside functions to make it safe to import this module.
+# Several imports are placed inside functions to make it safe to import this
```

Comment:
It looks like only 1 function is left. Do you understand the comment? It's not clear to me who is doing the importing of this module.